web/tests/functional/source_change/test_source_change.py

```diff
@@ -115,3 +115,46 @@ def test_store(self):
 
         ret = codechecker.store(self._codechecker_cfg, 'test_proj')
         self.assertEqual(ret, 1)
+
+    def test_store_config_file_mod(self):
+        """Storage should be successful if a non report related file changed.
+
+        Checking the modification time stamps should be done only for
+        the source files mentioned in the report plist files.
+        Modifying any non report related file should not prevent the storage
+        of the reports.
+        """
+        test_proj = os.path.join(self.test_workspace, 'test_proj')
+
+        ret = codechecker.log(self._codechecker_cfg, test_proj,
+                              clean_project=True)
+        self.assertEqual(ret, 0)
+
+        ret = codechecker.analyze(self._codechecker_cfg, test_proj)
+        self.assertEqual(ret, 0)
+
+        test_proj_path = self._testproject_data['project_path']
+
+        test_proj_files = os.listdir(self._codechecker_cfg['reportdir'])
+        for f in test_proj_files:
+            print(f)
+
+        # touch one file so it will be analyzed again
+        touch(os.path.join(test_proj_path, 'null_dereference.cpp'))
+
+        ret = codechecker.log(self._codechecker_cfg, test_proj)
+        self.assertEqual(ret, 0)
+
+        ret = codechecker.analyze(self._codechecker_cfg,
+                                  test_proj)
+        self.assertEqual(ret, 0)
+
+        test_proj_files = os.listdir(self._codechecker_cfg['reportdir'])
+        for f in test_proj_files:
+            print(f)
```

Comment:
I think this is only for debug purposes. If you would like to print these information please print something before the for loop which tells us what are these information:

```py
print("Content of the report directory:")
for f in test_proj_files:
    print(f)
```

or

```py
print("Content of the report directory: ", test_proj_files)
```

Reply:
I've just printed the information for debugging.